sdk/eventhub/event-hubs/README.md

```diff
@@ -436,24 +427,6 @@ export DEBUG=azure*,rhea*,-rhea:raw,-rhea:message
 export DEBUG=azure:*:(error|warning),rhea-promise:error,rhea:events,rhea:frames,rhea:io,rhea:flow
 ```
 
-### Logging to a file
```

Comment:
Just curious - why must this go?

Reply:
We had a README review this week and one of the things brought up was that for logging we should mostly refer the user back to the logger package README. In the case of event hubs we can mention DEBUG since it has been useful to get logs from rhea.